packages/design-system/src/hooks/use-tree/use-tree.ts

```diff
@@ -1,4 +1,4 @@
-import { isEqual } from 'lodash';
+import isEqual from 'lodash/isEqual';
```

Comment:
Is this better for tree-shaking? Just curious about the change.

Reply:
Somewhat of a temp fix as we investigate this error more closely which occurred in one of our consuming apps of the design system package

![image](https://github.com/user-attachments/assets/6b6e5d50-604c-4cd9-a50c-26ac5b595572)

Reply:
But without extra tooling, yes, it is technically better to do import like this for lodash for treeshaking purposes